docs/content/preview/manage/backup-restore/snapshot-ysql.md

```diff
@@ -31,79 +31,102 @@ showAsideToc: true
+Snapshots created using the `create_database_snapshot` never expire and are retained as long as the cluster exists. If a snapshot is no longer needed, you can delete it by running the [`delete_snapshot`](../../../admin/yb-admin/#delete-snapshot) command and providing the ID of the snapshot:
+
+```sh
+yb-admin delete_snapshot 0d4b4935-2c95-4523-95ab-9ead1e95e794
+```
+
+## Restoring a Snapshot
+
+To restore a database to a previously created snapshot, run the [`restore_snapshot`](../../../admin/yb-admin/#restore-snapshot) command and provide the ID of the snapshot:
 
-1. Get the current YSQL schema catalog version by running the [`yb-admin ysql_catalog_version`](../../../admin/yb-admin/#ysql-catalog-version) command:
+```sh
+yb-admin restore_snapshot 0d4b4935-2c95-4523-95ab-9ead1e95e794
+```
+
+The above command will rollback the database to the state which it had when the snapshot was created. The restore happens in-place, i.e. it changes the state of the existing database within the same cluster.
+
+## Moving a Snapshot to an External Storage
+
+Storing snapshots in-cluster is extermely efficient, but also comes with downsides. First of all, it increases the cost of the cluster - increasing number of snapshot can inflate the space consumption on the storage volumes. Second of all, in-cluster snapshots do not protect you from disaster scenarios like filesystem corruption or hardware failures.
+
+To mitigate the above, you might want to store backups outside of the cluster, in a cheaper storage that is also geografically separated from the cluster. This way, you can reduce the cost, and also restore your databases into a different cluster, potentially in a different location.
+
+To move a snapshot to an external storage, you need to gather all the relevant files from all the nodes, and copy then along with additional metadata that will be required when you decide to restore the snapshot on a different cluster. Below is the detailed step-by-step explanation of the process.
+
+1. Get the current YSQL schema catalog version by running the [`ysql_catalog_version`](../../../admin/yb-admin/#ysql-catalog-version) command:
 
     ```sh
-    yb-admin -master_addresses <ip1:7100,ip2:7100,ip3:7100> ysql_catalog_version
+    yb-admin ysql_catalog_version
     ```
 
-    The output displays the version:
-
     ```output
-    Version: 1
+    Version:1
     ```
 
-    If the version number here is greater than the version number from the first step, repeat all the steps to this point. The snapshot process can only guarantee consistency when no schema changes are made while taking the snapshot and the version number has remained the same.
+2. [Create an in-cluster snapshot](#creating-a-snapshot).
 
-1. Create the snapshot metadata file. Export the snapshot metadata using the following [`yb-admin export_snapshot`](../../../admin/yb-admin/#export-snapshot) command:
+3. Create a backup of the YSQL metadata using the [`ysql_dump`](../../../admin/ysql-dump) command:
```

Comment:
```suggestion
3. Back up the YSQL metadata using the [`ysql_dump`](../../../admin/ysql-dump) command:
```

Reply:
Don't we have a new approach to backups that doesn't involve dumps now? I thought we did...